tests/command.positionalOptions.test.js

```diff
@@ -161,7 +161,7 @@ describe('program with positionalOptions and default subcommand (called sub)', (
       .option('-s, --shared')
       .option('-d, --default')
       .action(() => {}); // Not used, but normal to have action handler on subcommand.
-    program.command('another'); // Not used, but normal to have more than one subcommand if have a default.
+    program.command('another'); // Not used, but normal to have more than one subcommand if it has a default.
```

Comment:
Not this one, I prefer the original.

Reply:
Sure, reverted. My grammar checker was complaining about a missing pronoun.

Reply:
'''
// Not used, but normal to have more than one subcommand if it has a default.
'''

The missing article it referring to the program, rather than the subcommand. So could do
```